pyproject.toml

```diff
@@ -109,6 +109,10 @@ dataflow = [
     "google-apitools",  # for beam pipeline
     "orjson==3.9.10",
 ]
+# Triton kernel dependency.
+triton = [
```

Comment:
maybe rename `triton` to `gpu` if this is a gpu specific dependency? Not sure if there will be other gpu only dependencies